aks/entrypoint.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 set -e
-export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION}"
+export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION} REPOSITORY_${GITHUB_REPOSITORY}"
 
```

Comment:
Do you want to keep similar pattern of using _ as delimiter. The pattern will be `GITHUBACTIONS_<actionName>_<repoName>`

Reply:
+1

Reply:
updated. thanks